control/velocity_controller/include/velocity_controller/delay_compensation.h

```diff
@@ -17,7 +17,7 @@
 #ifndef VELOCITY_CONTROLLER_DELAY_COMPENSATION
 #define VELOCITY_CONTROLLER_DELAY_COMPENSATION
 
-#include <autoware_planning_msgs/Trajectory.h>
+#include <autoware_planning_msgs/msg/trajectory.hpp>
```

Comment:
please reverse header order so it's from local to other pkgs to std includes

Reply:
Done. Thanks!